Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_5_Word Embedding_(D).md

```diff
@@ -0,0 +1,138 @@
+## Title : Name of the Topic
+🔴 **Type of Content** : Documentation
+
+🔴 **Domain** : Machine Learning
+
+🔴 **Module** : Name of the module for every week
```

Comment:
Add the module name. Module name the folder name or chapter name, under which you are adding your topic.

@Mann-tech13